filters/normalizepath/normalizepath.go

```diff
@@ -0,0 +1,38 @@
+package normalizepath
+
+import (
+	"strings"
+
+	"github.com/zalando/skipper/filters"
+)
+
+const (
+	Name = filters.NormalizePath
+)
+
+type normalizePath struct{}
+
+func NewNormalizePath() filters.Spec { return normalizePath{} }
+
+func (spec normalizePath) Name() string { return "normalizePath" }
+
+func (spec normalizePath) CreateFilter(config []interface{}) (filters.Filter, error) {
+	return normalizePath{}, nil
```

Comment:
I think you can `return spec, nil` and if you have a ptr receiver you don't need to have allocations for the filter:

```
func NewNormalizePath() filters.Spec { return &normalizePath{} }

func (f *normalizePath) Name() string { return filters.NormalizePathName }

func (f *normalizePath) CreateFilter(config []interface{}) (filters.Filter, error) {
    return f, nil
}

func (f *normalizePath) Request(ctx filters.FilterContext) {
...
```